test/example.js

```diff
@@ -1,22 +1,70 @@
 'use strict';
 
-const methodName = (
-  // Method caption
-  // First line description
-  // second line of multiline method description
-  a1, // number, comment
-  // multiline comment
-  // for first argument
-  a2, // array of string
-  a3 // comment
-  // Returns: string, comment
-  // Hint: message
-  // Example: methodName(5, [2, 3], 'f')
-  // Result: '52,3f'
-) => {
-  const s = a2.join();
-  return a1 + s + a3;
-};
+/* eslint-disable no-unused-vars */
+
+// One-liner function description
+// Multi-line expanded function description. Note that all lines should have
+// length of 80 or less characters
+//   num <number> argument description
+//   str - <string>, dash before and comma after types are optional
```

Comment:
Here you say that they are optional but at the bottom there is
> Current Metarhia style does not allow dash before and comma after types.

?

Reply:
They are indeed optional. I think that comments at the bottom should be removed from here and moved to the gist with our comments style because here it may sound confusing.